clientSrc/components/Header.js

```diff
@@ -11,9 +11,10 @@ const Header = () => {
             <div className='navbar-right'>
                 <a className="navbar-brand" href="login">Login</a>
```

Comment:
since you are using react router you should route with `<Link />` component of react router instead of html `a` tag

Reply:
got it, fixed